xbmc/windowing/WinSystem.h

```diff
@@ -25,10 +25,11 @@ struct RESOLUTION_WHR
 {
   int width;
   int height;
+  int screenWidth;
+  int screenHeight;
```

Comment:
The names are not very clear to me, "screen" is very generic. Going by their usage they seem to be logical dimensions  so maybe logicalWidth and logicalHeight? Either way please add a comment that they're measured in pixels. Not cm, in or some other uom.

It looks like you copied the surrounding style so maybe other rules apply, two nitpicks otherwise:
- the code guidelines ask for m_ prefix, and don't have an exception for structs - not sure if it's intentional.
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-structs, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#member-variables
- width or height cannot be negative - unsigned int seems more appropriate.

Reply:
@CrystalP screenwidth and screenheight are already defined on the `RESOLUTION` struct as ints, the `RESOLUTION_WHR` is only used to reduce the overall size of the information passed around. Changing this to unsigned int although most likely correct will require a lot of changes elsewhere (it's a 1:1 mapping) see https://github.com/xbmc/xbmc/blob/8f774d10a07b14de5dd9f9984c4e77058bc4e185/xbmc/windowing/Resolution.h#L87-L90.
I've added the pixel note to the documentation. Also changed to `m_` for members. Since in structs all members are public by default I find it a bit odd to access it using the `m_` prefix since most of the time those are private or protected.
Thanks for the review